package-lock.json

```diff
@@ -4,2241 +4,2951 @@
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "@babel/helper-plugin-utils": {
-      "version": "7.0.0",
-      "resolved": "https://registry.npmjs.org/@babel/helper-plugin-utils/-/helper-plugin-utils-7.0.0.tgz",
-      "integrity": "sha512-CYAOUCARwExnEixLdB6sDm2dIJ/YgEAKDM1MOeMeZu9Ld/bDgVo8aiWrXwcY7OBh+1Ea2uUcVRcxKk0GJvW7QA==",
-      "dev": true
+    "@babel/code-frame": {
+      "version": "7.8.3",
+      "resolved": "https://tools.adidas-group.com/artifactory/api/npm/npm-virtual/@babel/code-frame/-/code-frame-7.8.3.tgz",
```

Comment:
Watch out! Set your npmrc to public npm registry and generate package-lock again